pkgs/by-name/bo/bounty/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  libgit2,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "bounty";
+  version = "0.1.0-unstable-25-01-29";
+
+  src = fetchFromGitHub {
+    owner = "ghbountybot";
+    repo = "cli";
+    rev = "a1c8e74b1373ef7e0cc0cef158eeec59d56623fc";
+    hash = "sha256-sLbEGLvupDWk5FBdu4UZ4ohqMUK/HKAHhpB3oJwRA0c=";
+  };
+
+  cargoHash = "sha256-q+lnEpTsNFuQYQlQGoxZ5vnFUTqGWRMqhK+20H9Ke0I=";
+
+  doCheck = true;
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [
+    openssl
+    libgit2
+  ];
+
+  meta = with lib; {
```

Comment:
No `with` please except for the maintainers part

Reply:
done